inc/classes/cli/class-hmci.php

```diff
@@ -21,6 +21,13 @@ class HMCI extends \WP_CLI_Command {
 	 */
 	public static $progressbar;
 
+	/**
+	 * Progress ID to keep a unique value per runs, when threading
+	 *
+	 * @var string
+	 */
+	public static $progress_id;
```

Comment:
I'd call this thread_id

We're going to want to add a 2nd arg down the line 

(int) `thread_id`
and
(int) `threads` 
Will allow us to do auto paging for parallel threads